transitions/wave.h

```diff
@@ -73,6 +73,7 @@ class TrSparkX : public TransitionBaseX<SPARK_MS> {
 public:
 
   void run(BladeBase *blade) {
+    static_assert(!is_same_type<SPARK_MS, Int<0>>::value);
```

Comment:
There is no division by zero to worry about in this code, so this is not needed.